packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -244,6 +250,37 @@ ${renderCurrent(outdatedPkg)} ${chalk.grey('=>')} ${renderLatest(outdatedPkg)}`
     .join('\n\n') + '\n'
 }
 
+function renderOutdatedJSON (outdatedPackages: readonly OutdatedPackage[], opts: { long?: boolean }) {
+  const outdatedPackagesJSON = {} as {
+    [outdatedPackageName: string]: {
+      current: string
+      latest: string
+      dependencyKind?: 'dev' | 'optional'
```

Comment:
why is belongsTo not good? 

if you want to keep it, rename it to `dependencyType` and probably make it required.

Reply:
https://github.com/pnpm/pnpm/blob/main/packages/plugin-commands-outdated/src/outdated.ts#L278 I was going to fit to `renderPackageName`, but this is not a firm intention. fixed in 5abaceb